provisioning/device/src/ProvisioningDeviceClient.cs

```diff
@@ -79,12 +80,63 @@ public Task<DeviceRegistrationResult> RegisterAsync()
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The registration result.</returns>
         public Task<DeviceRegistrationResult> RegisterAsync(CancellationToken cancellationToken)
+        {
+            return RegisterAsync(default(RegisterInput), cancellationToken);
+        }
+
+        /// <summary>
+        /// Registers the current device using the Device Provisioning Service and assigns it to an IoT Hub.
+        /// </summary>
+        /// <param name="registerInput">The register input.</param>
+        /// <returns>The registration result.</returns>
+        public Task<DeviceRegistrationResult> RegisterAsync(RegisterInput registerInput)
+        {
```

Comment:
I don't follow what is Register Input ?

Reply:
I'll follow up with DPS team for more clarity here

Reply:
We do not need Register input here or anywhere. I'll remove it